modules/template/template.go

```diff
@@ -104,6 +105,15 @@ func NewFuncMap() []template.FuncMap {
 		"ThemeColorMetaTag": func() string {
 			return setting.ThemeColorMetaTag
 		},
+		"TabSizeClass": func(ec *editorconfig.Editorconfig, filename string) string {
+			if ec != nil {
+				def := ec.GetDefinitionForFilename(filename)
+				if def.TabWidth > 0 {
+					return fmt.Sprintf("tab-size-%d", def.TabWidth)
+				}
+			}
+			return "tab-size-8"
```

Comment:
While I would welcome such options, I think they'd be better placed in the user's settings panel instead of `app.ini`, because it's mostly a personal preference. Maybe through a new 'Display' settings pane.

Anyways, I'm fine if you want to keep it at 8. It's still the default in editors like `vim`, and I'm not really affected by it as i use `.editorconfig` in all my projects.